src/components/ComponentSettingsModal.jsx

```diff
@@ -208,7 +208,7 @@ const ComponentSettingsModal = ({
                   { isNewFile ? 'You may edit page details anytime. ' : ''}
                   To edit page content, simply click on the page title. <br/>
                   <span className={elementStyles.infoGrey}> 
-                    My workspace > Resources > { category } > <u className='ml-1'>{ title }</u><br/><br/>
+                    Resources > { category } > <u className='ml-1'>{ title }</u><br/><br/>
```

Comment:
Could we also deslugify category here?

Reply:
Good suggestions! I've applied the deslugifyDirectory to all Folders- and Resources- related components, d7160af.